priiloader/hacks_hash.ini

```diff
@@ -1,3 +1,41 @@
+[Block Disc Updates]

```

Comment:
whats with the overall shuffeling of hacks around?

Reply:
I reordered the hacks, as I said in another comment last month. 
I was thinking that hacks like "block online updates" or "block disc updates" and hacks like that are probably being used by way more people than obscure hacks like "run disc games under IOS249" or "No delete HAXX,DVDX...", and I reordered the hacks so that the hacks that I imagine are being used more often are at the top of the list, and the ones that aren't used that often are at the bottom / on the 2nd page. 
But if you don't like that I can change that back to how it was and just add the Wiimmfi patch somewhere.